snowshu/core/docker.py

```diff
@@ -101,8 +108,15 @@ def startup(self,  # noqa pylint: disable=too-many-locals, too-many-branches, to
         if is_incremental:
             name = self.replica_image_name_to_common_name(image_name)
             # get arch of the supplied image
-            base_image_arch = self.get_docker_image_attributes(image_name)[
-                'Architecture']
```

Comment:
you try to do the following:
`base_image_arch = Architecture[ self.get_docker_image_attributes(image_name)[
                'Architecture'].upper() ]` or via getattr